frontends/mit-open/src/common/utils.ts

```diff
@@ -0,0 +1,13 @@
+const getSearchParamMap = (urlParams: URLSearchParams) => {
+  const params: Record<string, string[] | string> = {}
+  for (const [key] of urlParams.entries()) {
+    const paramValues = urlParams.getAll(key)
+    const finalparams = paramValues.map((p) => {
+      return p.indexOf(",") !== -1 ? p.split(",") : p
+    })
+    params[key] = finalparams.flat()
```

Comment:
```suggestion
    const finalparams = paramValues.flatMap((p) => p.split(","))
    params[key] = finalparams
```

Main suggestion here is to always `p.split(",")`. Still behaves correctly if there's no comma.

Could also use `flatMap`.